<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NeoCD Image Splitter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTableWidget" name="tocTableWidget">
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QListWidget" name="logListWidget"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="loadCueButton">
       <property name="text">
        <string>Load CUE File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createSplitVersionButton">
       <property name="text">
        <string>Create Split File Version</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>loadCueButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>loadCUE()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createSplitVersionButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>createSplitFileVersion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadCUE()</slot>
  <slot>createSplitFileVersion()</slot>
  <slot>writeEquivalentNcd()</slot>
 </slots>
</ui>
